src/components/ChatLog.jsx

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+import messages from '../data/messages';
+
+const localUser = messages.length > 0 ? messages[0].sender : null;
+const ChatLog = ({ entries, toggleLike }) => {
+  return (
+    <div className="chat-log">
+      {entries.map((entry) => (
+        <ChatEntry
+          key={entry.id}
+          sender={entry.sender}
+          body={entry.body}
+          timeStamp={entry.timeStamp}
+          liked={entry.liked}
```

Comment:
Since the names of the keys on `entry` are the same as the names your are setting on `ChatEntry` attributes/you are using all of the values in `entry` you could do something something like this to save you a few keystrokes: 

```javascript
  const chatComponents = entries.map((entry) => {
    return(
      <ChatEntry
        {...entry}
        onLikeToggle={onLikeBtnToggle}
        key={entry.id}
      />
    );
  });
```